ROADMAP.md

```diff
@@ -1,54 +1,64 @@
 # Fluid Roadmap
 
-## Fluid 2024 Roadmap
+## Fluid 2025 Roadmap
 
-### Objective: Achieve orchestration of data operations and Kubernetes job scheduling systems
+### **1. Data Anyway**  
+**Objective**: Enable fluid data access **regardless of infrastructure constraints** (e.g., storage types, runtime environments) with minimal effort.
```

Comment:
Do we need to define what is the minimal effort here?

Reply:
Got it.